test/runners.jl

```diff
@@ -124,9 +124,13 @@ end
     end
 
     # This tests only that compilers for all platforms can build and link simple C code
-    @testset "Compilation - $(platform) - $(compiler)" for platform in platforms, compiler in ("cc", "gcc", "clang")
+    @testset "Compilation - $(platform) - $(compiler)" for platform in platforms, compiler in ("cc", "gcc", "clang"), version in (nothing, v"13")
```

Comment:
Sadly I don't think it's feasible to do this test with the current CI setup.

Reply:
That's a shame. Maybe we should setup this in buildkite? Maybe sharing the cache with Yggy?

Reply:
Yggdrasil doesn't have much cache, it's evicted every other day